prime_factor/prime_factor.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <limits.h>
+
+int generate_primes(int num) {
+    int cnt = 0;
+    printf("Prime Factors: ");
```

Comment:
Store it in an array that is passed as an argument to the function. Return the number of elements used in the array.